charts/kueue/templates/visibility/service.yaml

```diff
@@ -14,4 +14,5 @@ spec:
       targetPort: 8082
   selector:
   {{- include "kueue.selectorLabels" . | nindent 4 }}
+    control-plane: controller-manager
```

Comment:
we should not update the selectors

Reply:
Even if that was removed? 

Here: https://github.com/kubernetes-sigs/kueue/pull/1695/files#diff-af9ed29521052007a5babea215b30d063159b599410d819bee9ceebbc50f0ea8

And looks like it was generated by [hack/update-helm.sh](https://github.com/kubernetes-sigs/kueue/pull/1695/files#diff-1fa2d9febb7e8abaca7e772caeada182ec33aed9a3b3fa09f0e56775cf02336e)

Reply:
Removed